client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-source-site/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { Button } from '@automattic/components';
+import { StepContainer } from '@automattic/onboarding';
+import { useTranslate } from 'i18n-calypso';
+import DocumentHead from 'calypso/components/data/document-head';
+import FormattedHeader from 'calypso/components/formatted-header';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import type { Step } from '../../types';
+
+const SiteMigrationSource: Step = function ( { navigation } ) {
+	const translate = useTranslate();
+
+	const handleSubmit = () => {
+		navigation.submit?.();
+	};
+
+	const stepContent = (
```

Comment:
This content is pretty simple, we can use it directly on the prop, as the `formattedHeader`
Having it here can cause rerender on the component.

Reply:
I'm going to leave this as-is since the contents of the step will be changing as part of #87604.